pkg/action/install.go

```diff
@@ -63,6 +63,9 @@ const notesFileSuffix = "NOTES.txt"
 
 const defaultDirectoryPermission = 0755
 
+// Default timeout for CRD
+const defaultTimeout = 60 * time.Second
```

Comment:
```suggestion
// Minimum timeout for CRD installation (retained for compatibility reasons)
const minimumCRDTimeout = 60 * time.Second
```

nit: it isn't really the "default", it is the minimum. And only for CRDs. So the name should reflect that